admin/visitor_statistic_image.php

```diff
@@ -83,17 +83,17 @@
 $linie_still = [$grau, $grau, $weiss, $weiss];
 imagesetstyle($im, $linie_still);
 
-function teilen(&$item, $key, $teiler)
+function teilen(&$item, $teiler)
```

Comment:
Thats wrong!
teilen,multiplizieren and hinzufuegen are used as callback from array_walk. and it gets the item and the key from the array and a third parameter. so you can not remove $key